### PR TITLE
Survey/form navigation

### LINKED ISSUE #41: Add link to subscriptions from last survey page

### PR DESCRIPTION
**Pull request by amykarnaze**
### Type of change made:
- [ ] Bugfix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Testing - building tests
- [ ] Formatting - no new features
- [ ] Styling- no new features

### Detailed Description (Why is this change required? What problem does it solve? Any challenges that arose while implementing this feature? If so, how were the addressed?)

This PR adds more functionality to the survey form. A user can move forward and backward as they make style choices. The final submit page allows a user to view their subscriptions or reset the survey. 

###### Screenshots (if appropriate):

Screen Shot 2020-10-14 at 7 46 52 PM

###### Questions:

We have some pseudocode for our future endpoints.

### CODE REVIEW

File: code/web/src/modules/survey/api/actions.js [hunk 1]
```diff
@@ -40,5 +43,32 @@ export const moveBackward = () => (
...
+// export const getProducts = () => (
+//   dispatch => {
+//     dispatch({
+//       type: GET_PRODUCTS
+//       products
+//     })
+//   }
+// )
 
+// export const getImages = () => (
```
Comment (AurumValian) [inline code/web/src/modules/survey/api/actions.js:55]: If only the villainous CORS hadn't foiled our plans...

**Review by AurumValian — APPROVED** [review]
I'm excited! This is moving along well!

**Review by nickhartdev — APPROVED** [review]
Thanks for driving Amy! Excited to start tying things together

File: code/web/src/modules/survey/api/actions.js [hunk 2]
```diff
@@ -40,5 +43,32 @@ export const moveBackward = () => (
...
+//     return axios.get('https://randomfox.ca/images/')
+//       .then(response => {
+//         dispatch({
+//           type: GET_IMAGES,
+//           item: response.data
+//         })
+//       })
+//       .catch(error => console.log(error))
+//   }
+// )
```
Comment (Chulstro) [inline code/web/src/modules/survey/api/actions.js:66]: This is the endpoint request yes? Hopefully we get our side of this more complete for you guys very soon

**Review by Chulstro — APPROVED** [review]
Looking good!

File: code/web/src/modules/survey/api/actions.js (hunk 2 above)
Comment (nickhartdev) [reply code/web/src/modules/survey/api/actions.js:66]: Yeah, the endpoint will go in the place of the randomfox link! Appreciate y'all

**Pull request merged**